Sources/ArcGISToolkit/Components/Form/TextInputFooter.swift

```diff
@@ -54,13 +54,15 @@ struct TextInputFooter: View {
         currentLength: Int,
         isFocused: Bool,
         element: FieldFormElement,
-        input: FormInput
+        input: FormInput,
+        isRequired: Bool? = nil
     ) {
         self.currentLength = currentLength
         self.element = element
         self.isFocused = isFocused
         self.description = element.description
-        self.isRequired = element.isRequired
+        self.isRequired = (isRequired == nil ? element.isRequired : isRequired!)
```

Comment:
Parameter needs documentation:

`    ///   - isRequired: A Boolean value indicating whether the text input field is required.`